__tests__/core/queue.ts

```diff
@@ -677,6 +680,29 @@ describe("queue", () => {
           expect(key).not.toMatch(/workerA/);
         });
       });
+
+      test("retryStuckJobs", async () => {
+        queue.enqueue(specHelper.queue, "slowJob");
+        workerA.start();
+
+        await new Promise((resolve) => {
```

Comment:
You're already in an async context, why await a new Promise here?

Reply:
I want the test itself not to resolve until the promise is complete, since I'm using events here too.  that said, I can make it clearer by skipping the promise and the jest callback instead!

Reply:
simplified in https://github.com/actionhero/node-resque/pull/450/commits/c646bf7c1df0d788d26cd1630ca44e34012aeeb8